compiler/pragmas.nim

```diff
@@ -64,10 +64,10 @@ const
   varPragmas* = {wImportc, wExportc, wVolatile, wRegister, wThreadVar, wNodecl,
     wMagic, wHeader, wDeprecated, wCompilerProc, wCore, wDynlib, wExtern,
     wImportCpp, wImportObjC, wError, wNoInit, wCompileTime, wGlobal,
-    wGensym, wInject, wCodegenDecl, wGuard, wGoto, wExportNims, wUsed, wRaises}
+    wGensym, wInject, wCodegenDecl, wGuard, wGoto, wExportNims, wUsed}
```

Comment:
@Araq since wRaises is no longer valid for vars/lets/consts, the code will never reach https://github.com/nim-lang/Nim/blob/ceabbeadd7807dd21f274271acc779a7a8e69132/compiler/pragmas.nim#L1057-L1064
instead it will go to your new branch `elif comesFromPush and whichKeyword(ident) in {wTags, wRaises}:`, so yes, the code can be changed back.

Reply:
Created PR https://github.com/nim-lang/Nim/pull/10278